app/utils.py

```diff
@@ -75,6 +75,22 @@ def get_london_month_from_utc_column(column):
     )
 
 
+def get_london_year_from_utc_column(column):
+    """
+     Where queries need to count notifications by month it needs to be
+     the month in BST (British Summer Time).
+     The database stores all timestamps as UTC without the timezone.
+      - First set the timezone on created_at to UTC
+      - then convert the timezone to BST (or Europe/London)
+      - lastly truncate the datetime to month with which we can group
+        queries
+    """
```

Comment:
docstring still talking about month.

Also, I don't think we need to muck about with timezones here, because the year switch over is always in GMT +0. Think this can just be `func.date_trunc('month', column)` (and then there's a discussion about whether it needs its own function in utils, but i'll leave that decision with you